src/smart-components/group/add-group/add-group-wizard.js

```diff
@@ -50,36 +49,27 @@ const AddGroupWizard = ({
   };
 
   const steps = [
-    { name: 'General Information',
+    { name: 'General information',
       component: new GroupInformation(formData, handleChange, setIsGroupInfoValid),
       enableNext: isGroupInfoValid
     },
-    { name: 'Set Users',
-      component: new SetUsers(setGroupUsers, selectedUsers, setSelectedUsers,
-        optionIdx, setOptionIdx, createOption, handleChange)
+    { name: 'Add members',
+      component: new SetUsers({ formData, selectedUsers, setSelectedUsers, users })
     },
     {
-      name: 'Create policy',
-      steps: [
-        { name: 'Name and description',
-          component: new PolicyInformation({ title: 'Create policy (optional)',
-            formData, onHandleChange: handleChange, setIsPolicyInfoValid }),
-          enableNext: isPolicyInfoValid
-        },
-        { name: 'Add roles',
-          component: new PolicySetRoles({ formData, selectedRoles, setSelectedRoles, roles })
-        }
-      ]
+      name: 'Assign roles',
+      component: new SetRoles({ formData, selectedRoles, setSelectedRoles, roles })
     },
     { name: 'Review',
       component: new SummaryContent({ values: formData, selectedUsers, selectedRoles }),
       nextButtonText: 'Confirm',
-      enableNext: isGroupInfoValid && isPolicyInfoValid
+      enableNext: isGroupInfoValid
     }
   ];
   const fetchData = () => {
```

Comment:
I would have change this function completely
```JS
const  fetchData = () => {
 fetchGroup(id);
 fetchRoles();
 fetchUsers()
}
```
You are already connected to redux store so why not use it. You'll also want to add users and roles to mapped props at the top of component definiton. Also you can remove `setRoles` and `setUsers`